roles/apply_nmstate/tasks/main.yml

```diff
@@ -1,21 +1,22 @@
-- name: Install nmstate
-  package:
-    name: nmstate
-    state: present
-
-- name: "Copy rendered_nmstate_yml to {{ vm_nmstate_config_path }}"
-  copy:
-    content: "{{ rendered_nmstate_yml }}"
-    dest: "{{ vm_nmstate_config_path }}"
-    mode: 0644
-
-  # No commit is done to revert the changes if they cause the host to be come unreachable
-- name: Apply nmstate
-  shell: # noqa: no-changed-when
-    cmd: "nmstatectl apply --no-commit --timeout 120 {{ vm_nmstate_config_path }}"
-  async: 60
-  poll: 5
-
-- name: Commit changes
-  shell: # noqa: no-changed-when
-    cmd: "nmstatectl commit"
+---
+- name: Applying Nmstate
+  block:
```

Comment:
I'd like to see the `become: true` placed here (at the top of the block), similarly to how we do it in other parts of the code base, e.g. https://github.com/redhat-partner-solutions/crucible/blob/main/roles/create_vms/tasks/main.yml#L2-L13